dev-ops/docker/containers/app/Dockerfile

```diff
@@ -26,6 +26,12 @@ RUN sed -ri -e 's!VirtualHost \*:80!VirtualHost \*:8000!g' /opt/docker/etc/httpd
     && mkdir -p /usr/share/man/man1 \
     && curl -sL https://deb.nodesource.com/setup_11.x | bash \
     \
+    # Newer mysql-client
+    && sh -c 'echo "deb http://repo.mysql.com/apt/debian/ stretch mysql-8.0" >> /etc/apt/sources.list.d/mysql.list' \
```

Comment:
This would change the client from mariadb to mysql. That's problematic, because the dumps are not compatible.